microsoft/testsuites/kselftest/kselftest.py

```diff
@@ -209,7 +214,7 @@ def run_all(
 
         # get username
         username = self.node.tools[Whoami].get_username()
-        result_directory = f"/home/{username}"
+        result_directory = f"/tmp/{username}"
```

Comment:
keep the original path, and use a test case parameter to control the result path.